developers/wallets/native/index.md

```diff
@@ -219,3 +219,105 @@ object containing the following details:
 ```
 
 See the [OID4VCI Specification](https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html) for more details on this protocol.
+
+## Verifiable Credential Presentation
+
+Browser side verification flows will open the CHAPI wallet selector in the user's
+browser. If they select a native wallet, that URL will open the pre-registered
+native wallet apps link (see above for how to register...).
+
+That URL will to the Wallet's pre-registered domain--which will (thanks to the
+magic of app links!) open in the user's wallet:
+```
+https://wallet.example.com/switchboard?request=
+```
+
+Depending on the protocols supported, the value of request will contain one
+or more encoded JSON objects with nifty details...
+
+...explain when this happens...maybe with mermaid chart?
+
+### VC-API
+
+The `vcapi` property within the object parsed above will contain a URL the wallet can use to handle that request.
+
+To respond with one or more credentials via a VC-API exchanger, the wallet MUST first send a `POST` request
```

Comment:
And sometimes no credentials at all! (DID Authn only).

Reply:
@dlongley How about "zero or more"?

Reply:
"zero or more" works for me.